robottelo/entities.py

```diff
@@ -1156,60 +1156,54 @@ class Meta(object):
         server_modes = ('sat')
 
     def create_missing(self, auth=None):
-        """Customize the process of auto-generating instance attributes.
+        """Create a bogus managed host.
+
+        The exact set of attributes that are required varies depending on e.g.
```

Comment:
Yes. There are a variety of factors that determine exactly which set of attributes is required when creating a host, and the listed factors are a subset of all influencing factors. Another example of an influencing factor is that the `host[location_id]` attribute is required if organizations are enabled. I've added the "e.g." to indicate that the full list of influencing factors is not listed here.